lib/dealership.rb

```diff
@@ -0,0 +1,42 @@
+class Dealership
+
+  attr_reader :inventory
+  def initialize(name, address)
+  @name = name
+  @address = address
+  @inventory = []
+  @total_value = 0
```

Comment:
I probably wouldn't save this as an instance variable because you are only calculating and setting the value within the `details` method. Rather I would create a method for it and do the calculation in there.

```
def total_value
  total_value = 0
  @inventory.each do |car|
    total_value += car.total_cost
  end
  total_value
end
```